mmv1/third_party/terraform/tests/data_source_backup_dr_management_server_test.go

```diff
@@ -0,0 +1,71 @@
+package google
```

Comment:
we moved all tests to service folder recently. Would you mind moving this file to mmv1/third_party/terraform/services/backupdr as well?

Reply:
Moved in latest commit